src/common/paths/tasks_taskID_runs_runID_retry.yml

```
@@ -3,6 +3,11 @@ post:
   tags:
     - Tasks
   summary: Retry a task run
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: "../schemas/Run.yml"
```

```suggestion
        type: object
```

Unless `Run` is an actual valid body, I'd just specify `type: object` so the headers get set but no default body gets created. Using `type: object` results in the following typescript diff generated by oats (:

```sh
export GEN_TS=true
export DIFF_TS=true
make test-oats
```
```diff
--- orig-ts/common.ts
+++ ts/common.ts
@@ -7365,6 +7365,8 @@
   taskID: string
   runID: string
 
+  data?: any
+
   headers?: {
     'Zap-Trace-Span'?: string
   }
@@ -7393,7 +7395,10 @@
   request(
     'POST',
     `/api/v2/tasks/${params.taskID}/runs/${params.runID}/retry`,
-    params,
+    {
+      ...params,
+      headers: {...params.headers, 'Content-Type': 'application/json'},
+    },
     options
   ) as Promise<PostTasksRunsRetryResult>
 ```